coredocument/coredocument_test.go

```diff
@@ -25,6 +29,26 @@ var (
 	id5 = utils.RandomSlice(32)
 )
 
+var ctx = map[string]interface{}{}
+var cfg *config.Configuration
```

Comment:
Can we come up with a way to not to use a global ctx and cfg variables in places like these? I hope there is a way to pass some context to each test?

Reply:
I agree that we shouldn't use those globals in tests. I will try to remove them as part of the next PR, otherwise it might need to wait until we get rid of the bootstrapping logic.